src/components/content-tab-sync.tsx

```diff
@@ -78,7 +78,7 @@ function NoAuthSyncTab() {
 					</Button>
 				</Tooltip>
 			</div>
-			<div className="mt-3 w-[40ch] text-a8c-gray-70 a8c-body">
+			<div className="mt-3 max-w-[40ch] text-a8c-gray-70 a8c-body">
```

Comment:
```suggestion
			<div className="mt-3 text-a8c-gray-70 a8c-body">
```
Setting an explicit width on a text element where the width is already constrained by the parent seems odd to me. 

I think the better solution here is to drop this declaration.

Reply:
That's a great catch, Fredrik!

I couldn't find any justification for keeping the width there and the formatting looks good without it. Even when testing with different locales and screen sizes.

It couple be some leftover from the past. I could see some changes to the formatting when testing with the old code that was already replaced in https://github.com/Automattic/studio/pull/967.

Anyways, applying your suggestion.